PhysicsTools/NanoAOD/plugins/NanoAODOutputModule.cc

```diff
@@ -77,6 +77,7 @@ class NanoAODOutputModule : public edm::one::OutputModule<> {
   std::unique_ptr<TTree> m_tree, m_lumiTree, m_runTree, m_metaDataTree, m_parameterSetsTree;
 
   static constexpr int m_firstFlush{1000};
+  bool m_isPFScouting;
```

Comment:
we would rather avoid having such a content specific piece in the generic output module, for the sake of maintenance. Can you please explain why it cannot be done otherwise ?

Reply:
@alintulu any update on this?

Reply:
@clacaputo I've made a new implementation that does not use `m_isPFScouting `based on @vlimant and @missirol comments below. I was waiting to hear from @vlimant regarding another review comment before I update the PR. However I will push the new update now so it can be reviewed.